### PR TITLE
feat: add row_usages to ChunkProof struct

### PR DESCRIPTION
**Pull request by amoylan2**
### Purpose or design rationale of this PR

add row_usages to ChunkProof struct

### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [ ] build: Changes that affect the build system or external dependencies (example scopes: yarn, eslint, typescript)
- [ ] ci: Changes to our CI configuration files and scripts (example scopes: vercel, github, cypress)
- [ ] docs: Documentation-only changes
- [ ] feat: A new feature
- [ ] fix: A bug fix
- [ ] perf: A code change that improves performance
- [ ] refactor: A code change that doesn't fix a bug, or add a feature, or improves performance
- [ ] style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] test: Adding missing tests or correcting existing tests

### Deployment tag versioning

Has `tag` in `common/version.go` been updated or have you added `bump-version` label to this PR?

- [ ] No, this PR doesn't involve a new deployment, git tag, docker image tag
- [ ] Yes

### Breaking change label

Does this PR have the `breaking-change` label?

- [ ] No, this PR is not a breaking change
- [ ] Yes

### CODE REVIEW

File: common/types/message/message.go [hunk 1]
```diff
@@ -183,6 +183,12 @@ type ChunkInfo struct {
 	TxBytes       []byte      `json:"tx_bytes"`
 }
 
+// SubCircuitRowUsage tracing info added in v0.11.0rc8
+type SubCircuitRowUsage struct {
```
Comment (colinlyguo) [inline common/types/message/message.go:187]: we can replace this with l2geth's type.

Comment (lispc) [reply common/types/message/message.go:187]: I prefer current way.  First it is a super simple struct.  second Now ccc row usage and prover row usage are same type, in future we may have different details. So keeping 2 structs is not bad.

**Review by lispc — APPROVED** [review]

**Review by colinlyguo — APPROVED** [review]

File: common/types/message/message.go (hunk 1 above)
Comment (colinlyguo) [reply common/types/message/message.go:187]: makes sense.

**Comment by colinlyguo** [comment]
btw, we can merge this PR to develop branch and test the backward compatibility in sepolia. @amoylan2

**Pull request closed without merging**